### ISSUE TITLE
Adjust SVM hyperparameters

### ISSUE THREAD

**Issue by anikristo** [anikristo/DeCAF#10]
Target accuracy for SVM: ~83%
Target accuracy for SVM with Dropout: ~85% [I think we don't do Dropout right now]

NOTE: The paper uses 30 training examples per class, while we are using all. Since the dataset is a bit skewed (e.g. there are 55 training examples for accordion and 800 for airplanes), there may be a certain bias, which affects the overall accuracy. We need to adjust the preprocessor accordingly (#9).
